docs/rst/tutorials/cloud/wan_edge_cloud_tls/wan_edge_cloud_tls.rst

```diff
@@ -0,0 +1,390 @@
+.. _tutorials_cloud_wan_edge_cloud_tls_wan_edge_cloud_tls:
+
+
+Edge-Cloud TLS communication on WAN
+====================================
+
+.. warning::
+
+   In this example it is assumed the reader has basic knowledge of TLS concepts
```

Comment:
NIT:
This could be done with a glossary and adding entries to it, so if the user is interested could go to  brief description of each acronym